kano_settings/system/advanced.py

```diff
@@ -497,61 +497,63 @@ def set_dns_parental(enabled):
     refresh_resolvconf()
 
 
-def set_youtube_cookies(enabled):
+def set_everyone_youtube_cookies(enabled):
+
+    if get_parental_level() >= 2:
+        # Traverse through all users home directories to apply Midoris parental settings
+        username = []
+        try:
+            for username in os.listdir("/home/"):
+                set_user_youtube_cookies(enabled, username)
+        except:
+            logger.error('Error applying Midori security to users ({})'.format(','.join(username)))
+
```

Comment:
you need the `else` case, where you call `set_user_youtube_cookies(enabled, username)` with `enabled=False`